RubberduckTests/Refactoring/RenameTests.cs

```diff
@@ -2003,6 +2001,36 @@ MsgBox CStr(CranApple)
             tdo.MsgBox.Verify(m => m.NotifyWarn(It.IsAny<string>(), It.IsAny<string>()), Times.Never);
         }
 
+        [Test]
+        [Category("Refactorings")]
+        [Category("Rename")]
+        public void RenameRefactoring_RenameEnumerationMember_WarnMemberExists()
+        {
+            var tdo = new RenameTestsDataObject(selection: "Apple", newName: "Plum");
+            var moduleCode =
+                @"Option Explicit
+
+Public Enum FruitType
+    App|le = 1
+    Orange = 2
+    Plum = 3
+End Enum
+
+Sub DoSomething()
+    MsgBox CStr(Apple)
+End Sub";
+            var inputOutput = new RenameTestModuleDefinition("Module1", ComponentType.StandardModule)
+            {
+                Input = moduleCode,
+                Expected = moduleCode.Replace(FAUX_CURSOR,"")
+            };
```

Comment:
FYI this logic is now encapsulated in the `CodeString` class (causing problems with AutoCompletion, but very useful for tests); using it would remove the need to repeat this replacement everywhere.

Reply:
Incorporated `CodeString` into the `RenameTests`. BTW: There is a static function `CodeString.FromString` whose functionality is replicated in `CodeStringExtensions.ToCodeString`. (`CodeString.FromString` is not referenced) I'm wondering if the intent was for `CodeString.FromString` to disappear.(?)

Reply:
I think both will disappear with PR #4387.

Reply:
@MDoerner  to be clear, `"foo|".ToCodeString()` still works, except it will now returns a `TestCodeString`; the `CodeString` type (which is now a class) isn't dealing with any pseudo-caret pipe character anymore.